elastalert/schema.yaml

```diff
@@ -11,6 +11,17 @@ definitions:
     type: [string, array]
     items: {type: [string, array]}
 
+  timedelta: &timedelta
```

Comment:
`timeFrame` below includes a schedule property that is not supported by `datetime.timedelta`.  Keeping things strict